src/systems.rs

```diff
@@ -8,9 +8,9 @@ use bevy::{
         system::{Query, Res},
         world::World,
     },
-    prelude::Children,
```

Comment:
rustfmt did that! dunno why, if it wasn't correct before or smth. Anyhow tell me if you want this back.